http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/AppComponentSpec.scala

```diff
@@ -109,4 +109,22 @@ class AppComponentSpec extends AnyFlatSpecLike with TestComponent {
     }
   }
 
+  it should "get all apps for a given nodepool" in isolatedDbTest {
+    val savedCluster1 = makeKubeCluster(1).save()
+    val savedNodepool1 = makeNodepool(1, savedCluster1.id).save()
+    val savedNodepool2 = makeNodepool(2, savedCluster1.id).save()
+
+    val appName = AppName("test")
```

Comment:
I think you should rename it to something else, the unit test is complaining: `org.broadinstitute.dsde.workbench.leonardo.db.AppExistsException: An app with name test already exists for workspace 7ecd8890-2d17-4015-af2c-4d0c0d630499`